Assets/Scripts/Capabilities/Jump.cs

```diff
@@ -117,13 +128,10 @@ private void JumpAction()
             }
             if (!isGrounded && velocity.y < 0)
             {
-                velocity.y = jumpSpeed;
-            }
-            else
-            {
-                velocity.y += jumpSpeed;
+                // first zero out y velocity
```

Comment:
If I had to guess, that's probably what fixed the super jump. Thanks for ruining it >.>

Reply:
JK, it's still happening